lib/cluster.js

```diff
@@ -282,7 +283,7 @@ function masterInit() {
   function createWorkerProcess(id, env) {
     var workerEnv = util._extend({}, process.env);
     var execArgv = cluster.settings.execArgv.slice();
-    var debugPort = process.debugPort + id;
+    var debugPort = (process.debugPort + id) % debugPortRange + 1024;
```

Comment:
It would be a little neater to turn the number literal into a constant and use it in the range calculation above.

Aside, it's something of a (not consistently enforced) convention to name constants `kFoo`, i.e.:

```javascript
const kDebugPortStart = 1024;
const kDebugPortRange = (65536 - kDebugPortStart);
```